app/src/ui/branches/branch-list.tsx

```diff
@@ -91,6 +91,12 @@ interface IBranchListProps {
    * to prepopulate the new branch name field.
    */
   readonly onCreateNewBranch?: (name: string) => void
+
+  /** Can users perform operations on a branch in the list via a context menu? */
+  readonly canShowBranchContextMenu: boolean
+
+  readonly onCreateNewBranchFromStartPoint?: (branch: Branch) => void
```

Comment:
Could we combine this and `onCreateNewBranch`?

Reply:
Doesn't look like it because `onCreateBranch` is called from an `onClick` event so passes an event parameter when called from there.

https://github.com/desktop/desktop/blob/5e2a0e65013bc2d367c4e74409829f2d09447859/app/src/ui/branches/branch-list.tsx#L244